### PR TITLE
Scheduler

### PR DESCRIPTION
**Pull request by madhur4444**
Done - [Scheduler] Scaffold new service and setup basic framework to perform I/O w/ FB API #13
- Set basic service and framework
-  Interact with FB and post some random test text rn (To be replaced with text recv from Mantle) (WIP)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+start:
+	docker compose build && docker compose up
```
Comment (sid42) [inline Makefile:2]:
```suggestion
	docker compose up --build```

File: services/scheduler/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+gen:
+	python3 -m grpc_tools.protoc -I ../proto/ --python_out=../gen/ --grpc_python_out=../gen/ ../proto/scheduler.proto;
+	sed -i -E 's/^import.*_pb2/from . \0/' ../gen/*.py
```
Comment (sid42) [inline services/scheduler/Makefile:3]:
```suggestion
	sed -i -E 's/^import.*_pb2/from . \0/' ../gen/*.py
	
make gen-mac 
        python3 -m grpc_tools.protoc -I ../proto/ --python_out=../gen/ --grpc_python_out=../gen/ ../proto/scheduler.proto;
	sed -i "" -E 's/^import.*_pb2/from . \0/' ../gen/*.py
```

File: services/scheduler/Makefile [hunk 3]
```diff
@@ -1,6 +1,10 @@
 gen:
 	python3 -m grpc_tools.protoc -I ../proto/ --python_out=../gen/ --grpc_python_out=../gen/ ../proto/scheduler.proto;
 	sed -i -E 's/^import.*_pb2/from . \0/' ../gen/*.py
+	
+make gen-mac 
+        python3 -m grpc_tools.protoc -I ../proto/ --python_out=../gen/ --grpc_python_out=../gen/ ../proto/scheduler.proto;
+	gsed -i "" -E 's/^import.*_pb2/from . \0/' ../gen/*.py
```
Comment (sid42) [inline services/scheduler/Makefile:7]: should be 
``` 
gsed -i -E 's/^import.*_pb2/from . \0/' ../gen/*.py
```

Comment (Shreyas15srivatsa) [reply services/scheduler/Makefile]: Updated.

**Review by Shreyas15srivatsa — APPROVED** [review]
LGTM, updated PR to address @sid42's comments. Merging this in.

**Pull request merged**